README.md

```diff
@@ -7,20 +7,20 @@ This package implements the [libdns interfaces](https://github.com/libdns/libdns
 
 ## Authenticating
 
-This package supports **API Key authentication** but does not yet support **Sharing ID authentication**. Refer to the [LiveDNS documentation](https://doc.livedns.gandi.net/) for more information.
+This package only supports **API Key authentication**. Refer to the [Gandi's Public API documentation](https://api.gandi.net/docs/reference/#Authentication) for more information.
```

Comment:
Sharing ids are not an authentication mechanism. In fact a sharing id is the identifier of an organization.